glusterfsd/src/glusterfsd-mgmt.c

```diff
@@ -1597,6 +1597,8 @@ glusterfs_handle_node_status(rpcsvc_request_t *req)
 out:
     if (dict)
         dict_unref(dict);
+    if (output)
```

Comment:
Unrelated to this patch, but adding a check for valid `dict` & `output` after dict_new @1446 & @1534 would return immediately instead of doing some other operations and then failing.

Reply:
Took care of them in the latest commit.